### DISCUSSION TITLE
Ability to clear Reading List matches

### DISCUSSION THREAD

**Discussion by falo2k** [gotson/komga#1671] [Ideas]
When using matching with Reading Lists, if you get a "Duplicate Book" entry you're stuck unless you actively substitute another issue - there's no way to un-match the entry and leave it blank.  This can happen with some lists due to event overlaps, or issues that contain multiple stories that live in multiple places chronologically.  It would be good to offer the user the ability to ditch the duplicate entry and create the list using all remaining matches (much as you can choose to create a list with missing matches).

**Comment by gotson** [comment]
I've added the possibility to create a readlist even if duplicates are present, in which case the duplicates are omitted.

**Reply by falo2k** [reply]
Lovely, ta!

**Comment by gotson** [comment]
🎉 This issue has been resolved in `1.19.0` ([Release Notes](https://github.com/gotson/komga/releases/tag/1.19.0))
